src/mesh.rs

```diff
@@ -87,6 +88,16 @@ impl Mesh {
 }
 
 impl DynamicMesh {
+    /// Returns the indexed vertex position of the geometry base shape.
+    pub fn vertex(&self, index: usize) -> mint::Point3<f32> {
```

Comment:
Where is guarantee that we always can return value at any index? Need to use `Option` or smth else

Reply:
Agreed.